core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2982,6 +2995,10 @@ struct ViewValueFunctor<DeviceType, ValueType, true /* is_scalar */> {
                    size_t const arg_n, std::string arg_name)
       : space(arg_space), ptr(arg_ptr), n(arg_n), name(std::move(arg_name)) {}
 
+  ViewValueFunctor(ValueType* const arg_ptr, size_t const arg_n,
+                   std::string arg_name)
+      : space(ExecSpace{}), ptr(arg_ptr), n(arg_n), name(std::move(arg_name)) {}
+
```

Comment:
Missing initialization of `default_exec_space` here and in the preceding original constructor overload.

Reply:
That's part of #4823 (and fixed there).